modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryTopFivePodcastsView.kt

```diff
@@ -55,9 +55,9 @@ fun StoryFake1View(
             modifier = modifier.fillMaxWidth(),
             verticalArrangement = Arrangement.Center
         ) {
-            items(story.podcasts.size) { index ->
+            items(story.topPodcasts.size) { index ->
```

Comment:
Not a big deal, but I think we could simplify this just a bit with:

``` kotlin
import androidx.compose.foundation.lazy.items

// ...

items(story.topPodcasts) { topPodcast ->
    PodcastItem(
        podcast = topPodcast.toPodcast(),
        // ...
    )
}

```

Certainly not a blocker.

Reply:
Nice tip, applied in 793134d. Thank you!